app/src/androidTest/java/com/android/partagix/viewAccount/ViewAccountTest.kt

```diff
@@ -0,0 +1,376 @@
+  @Test
+  fun mainContentIsDisplayed() = run {
+    every { mockUserViewModel.uiState } returns emptyMockUiState
+    composeTestRule.setContent {
+      ViewAccount(
+          modifier = Modifier,
+          userViewModel = mockUserViewModel,
+          navigationActions = mockNavActions) // mockNavActions::navigateTo
+    }
+
+    onComposeScreen<ViewAccount>(composeTestRule) { mainContent { assertIsDisplayed() } }
+  }
+
+  //    userImage
+  @Test
+  fun userImageIsDisplayed() = run {
+    every { mockUserViewModel.uiState } returns emptyMockUiState
+    composeTestRule.setContent {
+      ViewAccount(
+          modifier = Modifier,
+          userViewModel = mockUserViewModel,
+          navigationActions = mockNavActions) // mockNavActions::navigateTo
+    }
+
+    onComposeScreen<ViewAccount>(composeTestRule) { userImage { assertIsDisplayed() } }
+  }
+
+  //    username
+  @Test
+  fun usernameIsDisplayed() = run {
+    every { mockUserViewModel.uiState } returns emptyMockUiState
+    composeTestRule.setContent {
+      ViewAccount(
+          modifier = Modifier,
+          userViewModel = mockUserViewModel,
+          navigationActions = mockNavActions) // mockNavActions::navigateTo
+    }
+
+    onComposeScreen<ViewAccount>(composeTestRule) { username { assertIsDisplayed() } }
+  }
+
+  // usernameText
+  @Test
+  fun usernameWorks() = run {
+    every { mockUserViewModel.uiState } returns emptyMockUiState
+    composeTestRule.setContent {
+      ViewAccount(
+          modifier = Modifier,
+          userViewModel = mockUserViewModel,
+          navigationActions = mockNavActions) // mockNavActions::navigateTo
+    }
+
+    onComposeScreen<ViewAccount>(composeTestRule) {
+      username { assertIsDisplayed() }
+      //            usernameText { assertIsDisplayed() }
+      val username = mockUserViewModel.uiState.value.user.name
+      usernameText { assertTextEquals("$username's profile") }
+    }
+  }
+
+  //    address
+  @Test
+  fun addressIsDisplayed() = run {
+    every { mockUserViewModel.uiState } returns emptyMockUiState
+    composeTestRule.setContent {
+      ViewAccount(
+          modifier = Modifier,
+          userViewModel = mockUserViewModel,
+          navigationActions = mockNavActions) // mockNavActions::navigateTo
+    }
+
+    onComposeScreen<ViewAccount>(composeTestRule) { address { assertIsDisplayed() } }
+  }
+
+  @Test
+  fun addressWorks() = run {
+    every { mockUserViewModel.uiState } returns emptyMockUiState
+    composeTestRule.setContent {
+      ViewAccount(
+          modifier = Modifier,
+          userViewModel = mockUserViewModel,
+          navigationActions = mockNavActions) // mockNavActions::navigateTo
+    }
+
+    onComposeScreen<ViewAccount>(composeTestRule) {
+      address { assertIsDisplayed() }
+      val userAddress = mockUserViewModel.uiState.value.user.address
+      //        address { assertValueEquals(userAddress)}
+      //        addressText { assertIsDisplayed() }
```

Comment:
I understand this is not yet implemented on the database. You could add a TODO flag so we don't forget to add those tests when we implement the Database support for user info.